src/tests/multihost/alltests/test_rfc2307.py

```diff
@@ -0,0 +1,93 @@
+""" Automation of rfc2307
+
+:requirement: rfc2307
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import random
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.libkrb5 import krb5srv
+from sssd.testlib.common.utils import SSHClient
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from constants import ds_instance_name, ds_suffix, ds_rootdn, ds_rootpw
+from sssd.testlib.common.exceptions import SSHLoginException
+from sssd.testlib.common.expect import pexpect_ssh
+
+
+def usr_grp(multihost, user_info, type):
+    ldap_uri = f'ldap://{multihost.master[0].sys_hostname}'
+    ds_rootdn = 'cn=Directory Manager'
+    ds_rootpw = 'Secret123'
+    ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+    krb = krb5srv(multihost.master[0], 'EXAMPLE.TEST')
+    unq_num = random.randint(9999, 999999)
+    if type == 'user':
+        usr = user_info.get('uid')
+        if ldap_inst.posix_user("ou=People", ds_suffix, user_info):
+            krb.add_principal(usr, 'user', 'Secret123')
+            return True
+        else:
+            print(f"Unable to add ldap User {user_info}")
+            assert False
+    if type == 'group':
+        try:
+            ldap_inst.posix_group("ou=Groups", ds_suffix, user_info,
+                                  memberUid=user_info.get('memberUid'))
+        except LdapException:
+            print(f"Unable to add ldap group {user_info}")
+            assert False
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.rfc2307
+class Testrfc2307(object):
+    """
+    This is test case class for ldap rfc2307
+
+        :setup:
+          1. Configure SSSD to authenticate against directory server
+          2. Enable debug_level to 9 in the 'nss', 'pam' and domain section
+    """
+    @pytest.mark.tier2
+    def test_0001_bz1362023(self, multihost, backupsssdconf):
+        """
+        :title: IDM-SSSD-TC: rfc2307: user with spaces at beginning
+        :id: 6923436c-d4e4-4a0d-a8f3-1e94ecb1dee3
+        :description: user with a white space at the beginning in it's name
+         should be able to log in
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1362023
```

Comment:
Hi,

this bugzilla ticket does not corresponds to what is tested, can you check if it is the right one?

bye,
Sumit

Reply:
Corrected the bugzilla link.